docs/_install-and-build/Installation-Instructions-for-Mac.md

```diff
@@ -25,7 +26,7 @@ You should now be able to run `geth` and connect to the network.
 
 Make sure to check the different options and commands with `geth --help`
 
-For options and patches, see: https://github.com/ethereum/homebrew-ethereum
+For options and patches, see: <https://github.com/ethereum/homebrew-ethereum>
```

Comment:
Why do you prefer urls in angle brackets? I don't mind either way, just asking.

Reply:
@adamschmideg This is done automatically by my markdown linter, for what markdown calls "naked" URLs, i.e. those without link text, this is the correct syntax, and turns them into clickable links, whereas, I think the former isn't.